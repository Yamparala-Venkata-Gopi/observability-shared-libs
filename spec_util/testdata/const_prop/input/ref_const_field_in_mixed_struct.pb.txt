# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    ref: {
      method_index: 1
      arg_ref: {
        key: "struct_mixed_arg"
        data_ref: {
          struct_ref: {
            field_ref: {
              key: "constant_field"
              data_ref: {
                primitive_ref: {}
              }
            }
          }
        }
      }
    }
  }
}
