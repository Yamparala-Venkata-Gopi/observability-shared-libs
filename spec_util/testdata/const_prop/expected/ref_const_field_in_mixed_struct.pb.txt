# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    value: {
      primitive: {
        type_hint: "string_constant_3"
        string_value: {
          value: "2b25d986-8dc5-45ea-aeb7-a01eb089af29"
        }
      }
    }
  }
}
